homeassistant/components/tellduslive/entry.py

```diff
@@ -119,7 +119,9 @@ def device_info(self):
         return {
             'identifiers': {('tellduslive', self.device.device_id)},
             'name': self.device.name,
-            'model': device['model'].title(),
-            'manufacturer': device['protocol'].title(),
+            'model':
+            device.get('model').title() if device.get('model') else None,
```

Comment:
Split this logic in more than one line an move it out of the dict. That will make it easier to read. Also, we shouldn't return the item in device_info if the value is `None`.
```py
device_info = {...}
model = device.get('model')
if model is not None:
    device_info['model'] = model.title()
...
```